src/main/java/org/gentoo/java/ebuilder/maven/MavenEbuilder.java

```diff
@@ -301,7 +301,7 @@ private void writeDependencies(final Config config,
                             mavenProject.getCommonDependencies(), null);
                 }
 
-                if (!mavenProject.getTargetVersion().isEmpty()) {
+                if (mavenProject.getTargetVersion()!=null) {
```

Comment:
in fact java arrays do not have isEmpty() method. in java, strings and collections have isEmpty() method for checking whether the string or collection is empty.